Discussion (#53)
thesis20/report: sections/futurework/futurework-chapter.tex

\section{Future work}\label{sec:futurework}

> In the following section we will define potential future work based on the information acquired.

Reviewer: ```suggestion
In the following section, we will define potential future work based on the information acquired.
```

> In the discussion, we highlighted multiple possible directions for future work.

Author: ref til discussion

> In the discussion, we highlighted multiple possible directions for future work.

Reviewer: ```suggestion
In the discussion (\autoref{sec:discussion}), we highlighted multiple possible directions for future work.
```

\subsection{Side-information to improve density}

> The \textit{IS-UserBased} methods shows potential for getting good results, but due to its item splitting approach, it creates a greater level of sparsity than already exists in the datasets.

Reviewer: ```suggestion
The \textit{IS-UserBased} method shows potential for getting good results, but due to its item splitting approach, it creates a greater level of sparsity than already exists in the datasets.
```

> The \textit{IS-UserBased} methods shows potential for getting good results, but due to its item splitting approach, it creates a greater level of sparsity than already exists in the datasets.

Reviewer 2: ```suggestion
The \textit{IS-UserBased} method shows potential for getting good results, but due to its item splitting approach, it creates a greater level of sparsity than there already exists in the datasets.
```

For future work on this method, adding side-information to items or users may help alleviate this sparsity and reconnect the fictional items it creates.
Adding this side-information to other methods such as \textit{NGCF} or \textit{LightGCN} may also be interesting to look into.

\subsection{Adding context to existing methods}
It could be interesting to look at ways to add context to \textit{LightGCN} or \textit{NGCF}.

> One way of doing this could be by employing methods to represent context in a format usable by the initial embeddings for these methods, and investigating results on the contextual datasets.

Author: måske nævn noget med at ændre deres convolutioon operationer til at tae hæjde for det
